.github/workflows/ci.yml

```diff
@@ -30,5 +30,5 @@ jobs:
           flags: unittests
       - name: Test oldest supported library versions
         run: gradle build -Pjackson.version=2.8.0 -Pspring.version=4.3.0.RELEASE -Pokhttp.version=3.3.0 -Papachehttp.version=4.3
-      - name: Test with latest library (micro) updates
```

Comment:
wouldn't we need another build for spring 5 then? Maybe we drop the spring 4 build?

Reply:
no,  actually we're compatible with all versions of spring with this one build.

Reply:
(and spring 5 is already integration tested in the gradle build run above)